<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>WildHealthDB - Wildlife Health Data Model</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./data_dictionary.html" rel="next">
<link href="./index.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./data_model.html">Wildlife Health Data Model</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">WildHealthDB</a> 
        <div class="sidebar-tools-main">
    <a href="https://github.com/dmontecino/WH_Database" rel="" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
    <a href="./WildHealthDB.pdf" rel="" title="Download PDF" class="quarto-navigation-tool px-1" aria-label="Download PDF"><i class="bi bi-file-pdf"></i></a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./data_model.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Wildlife Health Data Model</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./data_dictionary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Wildlife Health Data Model Dictionary</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./manual.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">WildlHealthDB Manual</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#main-units-of-the-data-model" id="toc-main-units-of-the-data-model" class="nav-link" data-scroll-target="#main-units-of-the-data-model">Main Units of the Data Model</a>
  <ul class="collapse">
  <li><a href="#project" id="toc-project" class="nav-link" data-scroll-target="#project">Project</a></li>
  <li><a href="#surveillance-activity" id="toc-surveillance-activity" class="nav-link" data-scroll-target="#surveillance-activity">Surveillance Activity</a></li>
  <li><a href="#outbreak-investigation" id="toc-outbreak-investigation" class="nav-link" data-scroll-target="#outbreak-investigation">Outbreak Investigation</a></li>
  <li><a href="#field-visit" id="toc-field-visit" class="nav-link" data-scroll-target="#field-visit">Field Visit</a></li>
  <li><a href="#location" id="toc-location" class="nav-link" data-scroll-target="#location">Location</a></li>
  <li><a href="#event" id="toc-event" class="nav-link" data-scroll-target="#event">Event</a></li>
  <li><a href="#collection" id="toc-collection" class="nav-link" data-scroll-target="#collection">Collection</a></li>
  <li><a href="#source-and-source-records" id="toc-source-and-source-records" class="nav-link" data-scroll-target="#source-and-source-records">Source and Source Records</a>
  <ul class="collapse">
  <li><a href="#generalities" id="toc-generalities" class="nav-link" data-scroll-target="#generalities">Generalities</a></li>
  <li><a href="#group-source" id="toc-group-source" class="nav-link" data-scroll-target="#group-source">Group Source</a></li>
  <li><a href="#animal-source" id="toc-animal-source" class="nav-link" data-scroll-target="#animal-source">Animal Source</a></li>
  <li><a href="#environmental-source" id="toc-environmental-source" class="nav-link" data-scroll-target="#environmental-source">Environmental Source</a></li>
  <li><a href="#arthropod-source" id="toc-arthropod-source" class="nav-link" data-scroll-target="#arthropod-source">Arthropod Source</a></li>
  </ul></li>
  <li><a href="#specimen" id="toc-specimen" class="nav-link" data-scroll-target="#specimen">Specimen</a>
  <ul class="collapse">
  <li><a href="#generalities-1" id="toc-generalities-1" class="nav-link" data-scroll-target="#generalities-1">Generalities</a></li>
  <li><a href="#specimens-of-group-source-records" id="toc-specimens-of-group-source-records" class="nav-link" data-scroll-target="#specimens-of-group-source-records">Specimens of Group Source Records</a></li>
  <li><a href="#specimens-of-animal-source-records" id="toc-specimens-of-animal-source-records" class="nav-link" data-scroll-target="#specimens-of-animal-source-records">Specimens of Animal Source Records</a></li>
  <li><a href="#specimens-from-environmental-source-records" id="toc-specimens-from-environmental-source-records" class="nav-link" data-scroll-target="#specimens-from-environmental-source-records">Specimens from Environmental Source Records</a></li>
  <li><a href="#specimens-from-arthropod-source-records" id="toc-specimens-from-arthropod-source-records" class="nav-link" data-scroll-target="#specimens-from-arthropod-source-records">Specimens from Arthropod Source Records</a></li>
  <li><a href="#specimens-with-a-mix-of-tissues-coming-from-a-unique-or-several-source-records" id="toc-specimens-with-a-mix-of-tissues-coming-from-a-unique-or-several-source-records" class="nav-link" data-scroll-target="#specimens-with-a-mix-of-tissues-coming-from-a-unique-or-several-source-records">Specimens with a mix of tissues coming from a unique or several Source Records</a></li>
  <li><a href="#specimens-from-diagnostic-products" id="toc-specimens-from-diagnostic-products" class="nav-link" data-scroll-target="#specimens-from-diagnostic-products">Specimens from Diagnostic Products</a></li>
  <li><a href="#pooled-specimens" id="toc-pooled-specimens" class="nav-link" data-scroll-target="#pooled-specimens">Pooled Specimens</a></li>
  <li><a href="#specimens-in-containers" id="toc-specimens-in-containers" class="nav-link" data-scroll-target="#specimens-in-containers">Specimens in Containers</a></li>
  </ul></li>
  <li><a href="#diagnostics" id="toc-diagnostics" class="nav-link" data-scroll-target="#diagnostics">Diagnostics</a></li>
  </ul></li>
  <li><a href="#complexities" id="toc-complexities" class="nav-link" data-scroll-target="#complexities">Complexities</a>
  <ul class="collapse">
  <li><a href="#laboratories" id="toc-laboratories" class="nav-link" data-scroll-target="#laboratories">Laboratories</a></li>
  <li><a href="#interpretation" id="toc-interpretation" class="nav-link" data-scroll-target="#interpretation">Interpretation</a></li>
  <li><a href="#clustering-of-locations-events-and-source-records" id="toc-clustering-of-locations-events-and-source-records" class="nav-link" data-scroll-target="#clustering-of-locations-events-and-source-records">Clustering of Locations, Events, and Source Records</a></li>
  <li><a href="#surveillance-activities" id="toc-surveillance-activities" class="nav-link" data-scroll-target="#surveillance-activities">Surveillance Activities</a></li>
  </ul></li>
  <li><a href="#all-units-in-the-data-model-and-all-relationships" id="toc-all-units-in-the-data-model-and-all-relationships" class="nav-link" data-scroll-target="#all-units-in-the-data-model-and-all-relationships">All Units in the Data Model and All Relationships</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content column-body" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Wildlife Health Data Model</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>The Wildlife Health Data Model (WHEDMO) is a data model that aims to accommodate wildlife health data coming from different sources, including local to national wildlife health surveillance systems (surveillance of diseases, pathogens, and toxins), research initiatives, or citizen-science based projects, among others.</p>
<p>Wildlife health surveillance systems can include a wide range of methods and targets. The plethora of surveillance types (general surveillance, targeted surveillance, participatory surveillance, sentinel surveillance, early warning surveillance, syndromic surveillance, etc) reflects the potential variability of the strategies, methodologies, and objectives associated with health assessment in wildlife. The strategies can include information from open sources, community members, and ranger patrols. Active capture of animals or the testing of specimen from wild animals in markets and collection of specimens of different types could be part of the strategies as well. The methodologies can focus in pure observations of herds and animals, hosts or vectors, the collection of carcasses to conduct necropsies, multiple diagnostic tests, among others. The objectives can range from early detection of pathogens, measure of disease levels and their impacts, prove absence of disease, evaluate disease control methods, etc. Furthermore, the variability with respect to the potential targeted populations and hazards (biological, chemical, physical, or physiological) support the creation of “infinite” types of data.</p>
<!-- # Figure data inputs -->
<p>On the other hand, harmonization of data and systems to effectively track, store, retrieve and disseminate information are foundational pillars of wildlife health surveillance systems. However, both pillars have being absent historically and globally. The challenge is evident. Promote best data practices when data are multiple and variable. The Wildlife Health Intelligence Network has develop a data model designed to respond to this challenge.</p>
<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>The WHE data model contains units that are hierarchical organized, starting from the highest level of “<strong>Projects</strong>”, followed by “<strong>Surveillance Activities</strong>”, “<strong>Field Visits</strong>”, “<strong>Locations</strong>”, “<strong>Events</strong>”, “<strong>Collections</strong>”, “<strong>Sources</strong>”, “<strong>Source Records</strong>”, “<strong>Specimens</strong>”, and “<strong>Diagnostics</strong>”. Briefly, <strong>Projects</strong> are a set of <strong>Surveillance Activities</strong> with a common leader and organizer (e.g., a national wildlife health surveillance system). <strong>Surveillance Activity</strong> is a surveillance task that have a specific methodology and objectives. <strong>Field Visit</strong> represents a period of time when activities are conducted in the field. The <strong>Location</strong> is a specific area where surveillance activities are conducted. An <strong>Event</strong> is a site where wildlife health surveillance data is collected from. An <strong>Event</strong> can contain different <strong>Collections</strong> that represent different methods and efforts to obtain <strong>Source Records from Sources</strong>. <strong>Sources</strong> are the units that can provide <strong>Specimens</strong> for hazard detection or whose health is assessed by an agent at time <em>t</em>. There are four Source types in the data model: group of animals, individual animals, sites to collect environmental tissue, and sites to collect arthropods. <strong>Specimens</strong> are tissue taken from the <strong>Sources</strong> at time <em>t</em> (<strong>Source Records</strong>). <strong>Diagnostics</strong> are the tests conducted in lived animals (“Animal Source”), <strong>Carcasses</strong>, or <strong>Specimens</strong> collected to assess hazards.</p>
<p>Out of this hierarchy is the component <strong>Diagnosis</strong>, which is the final status assigned to a Diagnostic, a Specimen, or a Source at time <em>t</em> with respect to the hazards surveyed, based on well-documented case definitions. <strong>The basic relationships among the basic units of the data model</strong> are shown in Figure 1 below.</p>
<p><img src="Pictures_and_diagrams/relationship_data_model.png" class="img-fluid" style="width:120.0%"></p>
<p><strong>Figure 1</strong>: Basic relationships among the basic units of the data model.</p>
<p>The data model remains simple when the wildlife health data is simple, and it gets more complicated as the data structure and variability increases. For this reason, many components of the data model are conditional, meaning that they are needed depending on the structure of the data. Let’s start with example 1: wildlife mortality reported by a community member through an application. The methodology must be explained in the Surveillance Activity metadata. The Field Visit could be one per year to categorize the reports per year. The Location can be defined by the position sent by the mobile device when each dead animal is recorded. In this case, it could be a county of a zip code. In the example, an Event has Source Records corresponding to dead animals. No Specimen, Diagnostic, or Laboratory data is generated. No Collection effort is associated with the Events, and the animals are not clustered beyond the Field Visit, the Location, and the Events.</p>
<p><img src="Pictures_and_diagrams/simple_example.png" class="img-fluid" style="width:80.0%"></p>
<p><strong>Figure 2</strong>: Components of the data model to contain data from example 1.</p>
<!-- The next example is a series of surveys in wildlife farms to identify pathogens. Surveys are conducted every 3 months and each one of them include all the farms in the study. The farms are distributed in several counties. Each farm has several species that are distributed in barns, aisles, pens, and cages. Seasonal changes are assessed as well. All the animals of the cages will be sampled (rectal and oral swab each one). Fecal samples will be collected from the cages directly as well. Finally, each sample will be tested for 3 pathogens. In this example, there could be a single Surveillance Activity with the three pathogens included, or three Surveillance Activities with the same methods except the targeted hazards (one Surveillance Activity per pathogen is recommended). The data includes **Field Visits every three months**. Each one of these Field Visits includes all the counties were the farms are located. If each farm is a Location, then the Locations must be clustered by county. Also, if the Event within each farm is a cage with animals, then the Events must be clustered by pen, aisles, and barns. In the manner the structure of the data is Field Visit, county (cluster), farms (Location), barns (cluster), aisles (cluster), pens (cluster), and cages (Event). Depending on the number of cages per pen, there could be multiple or a unique Event per pen. Each Event will have as many Sources at time *t* as animals are per each cage. Continuing with the animals (Sources), two Specimens are collected from each one of them and they are tested for three pathogens for a total of six tests per individual. Furthermore, each cage Event also has a Group Source Record, corresponding to the animals that are in the cage as a group. This Group Source Record has a fecal Specimen associated also tested for three pathogens. -->
<!-- # Figure first exaple -->
<p>The data model is <strong>not designed for wildlife population monitoring</strong>. However, the data model includes key identifiers so wildlife health and population data can be integrated seamlessly.</p>
<p>More complex relationships among the basic units in the data model, as well as other extra information in the data model are explained in the section “Complexities”. It is recommended first to learn more about the basic units from Project up to Diagnosis by continuing reading this documentation and checking the menu on the top right corner of this website and click on the corresponding unit -&gt;.</p>
</section>
<section id="main-units-of-the-data-model" class="level1">
<h1>Main Units of the Data Model</h1>
<section id="project" class="level2">
<h2 class="anchored" data-anchor-id="project">Project</h2>
<!-- # Project {.unnumbered} -->
<p>A Project in the data model is a surveillance initiative supported by specific entities. For example, the PREDICT Project funded by USAID, an isolated cross-sectional study with a single field visit to a single location (sample collection in a market), or a national or local wildlife health surveillance network led by a government agency. Projects are the highest hierarchical unit in the database and they contain at least one Surveillance Activity. Projects can involve a single date or they can extend for as long as needed.</p>
<p>Properties of a Project include the Project ID, Project Code, Project Cross Reference ID, Project Cross Reference Origin, Project Leader, Project Funding Source, among others (see Data Dictionary).</p>
</section>
<section id="surveillance-activity" class="level2">
<h2 class="anchored" data-anchor-id="surveillance-activity">Surveillance Activity</h2>
<p><img src="Pictures_and_diagrams/Picture3.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Surveillance Activity {.unnumbered} -->
<p>A Surveillance Activity in the data model refers to a set of activities to collect data with the aim of detecting, proving freedom of disease, measure incidence or prevalence, or assessing trends of specific health hazards within specific populations, using a common methodology. For example, the <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274490">longitudinal assessment of coronavirus shedding in two <em>Eidolon helvum</em> bat roosts in Africa</a> collected a fix number of feces from two bat roosts of the same species on a monthly basis for 12 months and tested the feces for coronaviridae sp.</p>
<p>Surveillance Activities can encompass targeted surveillance, scanning surveillance, citizen science monitoring, ranger patrols, and others. <strong>Surveillance Activities include detailed metadata that describe their objectives and methods, including information on the targeted species, populations, hazards, samples, collection methods, diagnostics, and case definitions, among others</strong>. A Surveillance Activity can include multiple Field Visits and Locations and Source types (groups, individual animals, environmental sources, or arthropod collections).</p>
<p>It is recommended that the definition of Event, what an Event represents, is unique for the Surveillance Activity and also to keep a minimum number of hazards targeted under the same Surveillance Activity. In general, a specific set of methods, strategies, and objectives should equal a specific Surveillance Activity.</p>
<p><strong>Field Visits, Locations, Events, Sources, Source Records, and Diagnostics usually belong to a single Surveillance Activity</strong>. This is the Surveillance Activity that lead to the Field Visits at different Locations to document Events, collect Sources and Specimens, perform Diagnostics for a specific hazard, and provide Diagnosis for Diagnostics, Specimens, and Sources. However, there are exceptions to this rule (See the “Complexities” section below). Furthermore, <strong>a Surveillance Activity usually includes Field Visits, Locations, Events, Sources, Source Records, and Diagnostics, but this is not always the case</strong> (See the “Complexities” section below).</p>
<p>Most of the properties of a Surveillance Activity are associated with the standard documentation of the methods use to conduct the corresponding surveillance initiative. These properties include, Start Date, End Date, Hazards and Taxa targeted, Organizations involved, the definition of Location, the definition of Event, the type of Sources included, the methods to find or establish Sources, Specimens collected and method of collection, the diagnostics used, case definitions for the diagnostic tests, the Specimens, and the Sources at time <em>t</em>, among others.</p>
</section>
<section id="outbreak-investigation" class="level2">
<h2 class="anchored" data-anchor-id="outbreak-investigation">Outbreak Investigation</h2>
<p>Each outbreak investigation is considered a Surveillance Activity. Outbreak Surveillance Activities can encompass the Field Visit that lead to the Outbreak detection and the following Field Visits associated with Outbreak investigation and control. The detection could have been made during a Field Activity conducted by rangers during their patrols. This initial Field Visit and its components should be assigned to a the ranger patrol Surveillance Activity and to the Outbreak Surveillance Activity. The following Field Activities that could be completed by veterinarians or public health officers conducting the official outbreak investigation receive the outbreak Surveillance Activity only (See Complexities below).</p>
</section>
<section id="field-visit" class="level2">
<h2 class="anchored" data-anchor-id="field-visit">Field Visit</h2>
<p><img src="Pictures_and_diagrams/Picture4.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Field Visit {.unnumbered} -->
<p>A Field Visit in the data model represents a time period, including a start and end date, to visit Locations, identify Events and Sources, collect Specimens (see below) and document data from them. A Field Visit can include various types of Locations (see below), such as markets, natural areas, rehabilitation centers, caves, etc. that are visited within the same trip and common start and end date. A Field Visit should include at least one Location up to an indefinite number of Locations (areas).</p>
<p>Properties of a Field Visit include the Start and End Date, Field Visit ID, Field Visit Code, Field Visit Cross Reference ID, Field Visit Cross Reference Origin, Leader of the Visit, among others (see Data Dictionary). If the data of interest per Field Activity includes attributes that are not part of the data model, it has to be reported as part of the Surveillance Activity metadata and it is recommended to keep track of these attributes in a different source (another database, an excel sheet, etc.). Common extra Attributes can be added to the data model in the future. Missing Options for single- and multi-selection attributes of Field Visits can be added as long as they promote a controlled vocabulary.</p>
</section>
<section id="location" class="level2">
<h2 class="anchored" data-anchor-id="location">Location</h2>
<p><img src="Pictures_and_diagrams/Picture5.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Location {.unnumbered} -->
<p>A Location in the data model refers to a general area (polygon) where Events can be identified, and Source Records, Carcasses, and Specimens can be collected from. A Location serves as a way to group data in areas compared to the Events that group data under a exact latitude and longitude (‘Event’; see below). A single Location (area) can include between zero Events (e.g., a Location is explored but no Event is found) up to a indefinite number of Events (points).</p>
<p>What a ‘Location’ represents will inevitably vary depending on the Surveillance Activity methodology. For example, for the collection of arthropods, a Location can represent a parcel where traps are set. In a structured/hierarchical/clustered/nested research, the Location could represent a grid cell. In ranger patrols, the Location can correspond to either the full protected area, or a specific zone within the protected area. <strong>It is up to the user to define what unit the Location area represents and report it in the Surveillance Activity metadata</strong>.</p>
<p>Properties of Location include Location ID, Location Code, Location Cross Reference ID, Location Cross Reference Origin, Location Type, Environmental Characteristics, among others (see Data Dictionary). If the data of interest per Location includes attributes that are not part of the data model, it has to be reported as part of the Surveillance Activity metadata and it is recommended to keep track of these attributes in a different source (another database, an excel sheet, etc.). Common extra Attributes can be added to the data model in the future. Missing Options for single- and multi-selection attributes of Locations can be added as long as they promote a controlled vocabulary.</p>
</section>
<section id="event" class="level2">
<h2 class="anchored" data-anchor-id="event">Event</h2>
<p><img src="Pictures_and_diagrams/Picture6.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Event {.unnumbered} -->
<p>An Event in the data model refers to a distinct wildlife health event that occurs at a specific longitude and latitude on a particular date (time stamp). A single Event (point) can include between zero Collections (e.g., opportunistic detection of Sources) up to a indefinite number of Collections.</p>
<p>The definition of a wildlife health event, and thus what an ‘Event’ represents, will inevitably vary depending on the Surveillance Activity. For example, in active surveillance efforts, an Event can refer to the site where animals are captured, starting at time <em>t</em>, to obtain Specimens, the site and time where a dead dead animal is found during a ranger patrol, the site and time where water is collected from a pond. In the case of surveillance for vector-borne diseases, an Event can represent a site where one or several traps for vectors are deployed at time X. In the case of beached fish, an Event can represent the position and time of recording of each individual dead fish in one extreme, or the total count of dead fish across the beach reported as a single point in the other extreme. In the context of a wet market, the definition of Event can be applied to the market, to vendors, to the stalls of vendors, or to the cages in the stalls at time <em>t</em>. For study A the Event could the grid cell where traps are deployed, whilst for study B it could each trap within a grid cell.</p>
<p>Moreover, the presence of at least one Source could be needed to establish an Event. For example, a dead animal found during a ranger patrol at time <em>t</em> leads to an Event. In Active Surveillance, an <strong>Event might not have any Source, for example, a capturing effort starting at time <em>t</em> that ends with no animals captured</strong>. Additionally, the definition of an Event can include or not healthy animals. In the example of the dead animal found by the ranger at time <em>t</em>, it could be possible that the Event definition includes the documentation of healthy animals or not.</p>
<p>Consequently, an Event in wildlife health surveillance can represent different entities depending on the objectives and the methodology used to collect wildlife health data. <strong>It is up to the user to define what unit the Event represents and report it in the Surveillance Activity metadata. It is recommended that each Surveillance Activity has a single definition of what an Event represents</strong>.</p>
<p>Properties of Event include the Start Date, Event ID, Event Code, Event Cross Reference ID, Event Cross Reference Origin, Longitude, Latitude, the Coordinate Reference System used, among others (see Data Dictionary). If the data of interest per Event includes attributes that are not part of the data model, it has to be reported as part of the Surveillance Activity metadata and it is recommended to keep track of these attributes in a different source (another database, an excel sheet, etc.). Common extra Attributes can be added to the data model in future versions. Missing Options for single- and multi-selection attributes of Events can be added as long as they promote a controlled vocabulary.</p>
</section>
<section id="collection" class="level2">
<h2 class="anchored" data-anchor-id="collection">Collection</h2>
<p><img src="Pictures_and_diagrams/Picture7.png" class="img-fluid" style="width:100.0%"></p>
<p>A <strong>Collection</strong> in the data model represents the effort associated with the observation, finding, capturing, etc. of Sources at the longitude and latitude of an Event that starts at the Event time stamp. For example, the number of hours an observer spent in a post to identify dead birds in a wetland and the tools used to find it (e.g., telescope, binoculars); the number of camera traps in a specific latitude and longitude to photograph a sick animal and the hours they were deployed; the distance and time traveled by a ranger to find a dead animal; the type of trap, bait, time period, and set of equivalent traps to collect mosquitoes at a specific site.</p>
<p><strong>A Collection can contain between zero Source Records (e.g., a capturing effort but no animal is captured) up to a indefinite number of Source Records</strong>. If Sources are found opportunistically, then no Collection is involved (e.g., articles from news articles where the coordinates of the site and the date reported determine the Event and the animals involved are the Source Records).</p>
<p><strong>A Collection usually contains a specific type of Source Record (either Group Source Record, Animal Source Record, Environmental Source Record, and Invertebrate Source Records)</strong>. This is because observation, capturing, or other efforts are usually targeting groups of animals, or individual animals, or environmental tissue, or arthropods. On the other hand, because multiple Collections, targeting any of these objectives, can be part the same Event (e.g., traps to collect mosquitoes and also use a device to filter and collect air at the same site), then an Event can contain Source Records of multiple types.</p>
<p><strong>Collections are defined by the type of spatial and temporal units to complete it, the number of those units, and the position of those number of units with respect to the spatio-temporal coordinates of the Event</strong>. A spatial unit can be mist nets deployed, CO2 traps deployed, camera traps deployed, kilometers walked, square kilometers scanned, etc. A temporal unit can be hours walking, days searched, minutes observed, etc. Then, each Collection has a number of the assigned unit (e.g., “6” for number and “kilometers walking” for unit). Finally, the position with respect to the Event spatio-temporal coordinates can be “at the Event”, “around the Event”, “from the beginning of the Event”, “up to the Event”, etc.</p>
<p>For example, a Collection of an Event constituted by an individual at a post in a wetland that spent 6 hours to scan for dead birds using binoculars has as spatial Attributes: “1”, “observer binocular” “at the Event”; and temporal Attributes: “6”, “hours”, “at the Event”.</p>
<p>Another example is a ranger walking for 10 kilometers since the beginning of the patrol to find a dead animal. The spatial Attributes are: “10”, “kilometers walked”, “to the Event”; and temporal Attributes are not relevant (or the time walking to get to the Event could be reported as well). Similar approach can be used to record animals found during an outbreak investigation. A final example is two CO2 traps and a BG trap to collect mosquitoes deployed for a night. In this case, there are two Collections as <strong>a set of equivalent efforts constitute a single Collection per Event</strong>. The spatial Attributes of the CO2 traps are: “2”, “CO2 traps”, “at the Event”; and temporal Attributes are: “1”, “night”, “at the Event”. The spatial Attributes of the BG trap are: “1”, “BG trap”, “at the Event”; and temporal Attributes are: “1”, “night”, “at the Event”. To document the three traps separately, three Events with a single trap each must be created.</p>
<p><strong>It is not accepted to provide the spatial relationship with the Event as “meters after the Event” or the time relationship with the Event as “hours before the Event”</strong>.</p>
<p>Collections also allow to characterize problems during the search for Sources, such as camera traps running out of battery or stolen, torn mist nets, etc.</p>
<p>When Source Records are obtained without a <strong>Collection</strong> (e.g., animals sampled in a market, citizen science reports of dead animals) the property “Unit” is “none”, the “number of units” is NA, and the positioning with respect to the Event is NA.</p>
<p>Further properties of Collection include the Collection ID, Collection Code, Collection Cross Reference ID, Collection Cross Reference Origin, Problems, among others (see Data Dictionary). If the data of interest per Collection includes attributes that are not part of the data model, it has to be reported as part of the Surveillance Activity metadata and it is recommended to keep track of these attributes in a different source (another database, an excel sheet, etc.). Common extra Attributes can be added to the data model in future versions. Missing Options for single- and multi-selection attributes of Collection can be added as long as they promote a controlled vocabulary.</p>
</section>
<section id="source-and-source-records" class="level2">
<h2 class="anchored" data-anchor-id="source-and-source-records">Source and Source Records</h2>
<p><img src="Pictures_and_diagrams/Picture8.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Source and Source Records {.unnumbered} -->
<section id="generalities" class="level3">
<h3 class="anchored" data-anchor-id="generalities">Generalities</h3>
<p>A Source in the data model is a unit that can be observed and provide Specimens, including carcasses, organic tissue, abiotic tissue, or arthropods. Sources can provide between zero (no sampling) up to indefinite number of Specimens (tissue to conduct diagnostic tests). The data model manages four types of Sources: i) a co-specific group of animals (Group Source), ii) individual animals (Animal Source), iii) sites that can provide abiotic tissue of interest (e.g., sediment, soil, water, air) or biotic material of animal origin whose individual or group animal of origin is unknown (Environmental Source, e.g., feces found in the field), and iv) sites where active or passive collection arthropods at any life stage is completed (Arthropod Source). A Group Source could correspond to a specific bat roost of species X sourcing guano, an Animal Source could be a collared animal, an Environmental Source could be a pond where water is collected from, and an Arthropod Source could be a site where mosquito larvae are collected from. Sources contain time-independent data only, such as the species of an animal in the case of a Group Source or an Animal Source (more below).</p>
<p><strong>All Sources can potentially be individually identified and followed over time if needed</strong>. Environmental and Arthropod Sources are associated with a specific site making it easy to track them over time; however, <strong>Animal Sources and Group Sources might not be individually identified preventing following them over time in new Collections or an opportunistic finding</strong>.</p>
<p><strong>A Source observed, collected, or captured at time <em>t</em> can be linked to Events through a Collection (see previous section) via a Source Record (the Source at time <em>t</em>)</strong>. These Source Records contain time-dependent data only, such as health status at time <em>t</em> (Group and Animal Sources). <strong>Sources individually identified can be followed longitudinally and have multiple Source Records that link them to multiple Events over time of the same or different Surveillance Activity</strong>. For example, a bat (Source) caught in a mist net (Collection) set up at a roost (Event), identified individually, and recaptured in a following Event. This bat is linked to the capture Event and the recapture Event through a single Source Record for each Event.</p>
<p><strong>Sources that are not individually identified will be linked as a Source Record to the Event where they were observed or captured only</strong>. For example, a bat (Source) that is captured (Collection) and sampled during an Event but not marked or a dead animal not identified found by a ranger during a patrol. The bat and the animal found are linked only to the Event they were capture and observed, respectively.</p>
<p><strong>Source Records found opportunistically can be linked to an Event in the absence of a Collection</strong>. For example, a dead animal (Source Record) reported by a citizen who found it in a beach (Event).</p>
</section>
<section id="group-source" class="level3">
<h3 class="anchored" data-anchor-id="group-source">Group Source</h3>
<p><strong>A Group Source is a unit of co-specific individuals (animals of the same species) associated to a herd, an area, a site, a farm, a cage, a stall, an enclosure, or other that makes the individuals part a single epidemiological unit</strong>. Group Sources can be observed, captured, and provide Specimens at time <em>t</em> as a collective. Data of a Group Source include the Species, the Group Source ID, the Group Source Cross Reference ID, among others (see Data Dictionary).</p>
<p>The purpose of Group Source is to record individuals at the species level instead of individual by individual. This can occur when herds are the unit of interest, when protected area rangers find animals of the same species in a health event, or to document animals in a unit such as a cage, stall, or vendor of a market. For example, animals of two species in a single cage, not tracked individually, represent two Group Sources (one Group Source by species).</p>
<section id="group-source-record" class="level4">
<h4 class="anchored" data-anchor-id="group-source-record">Group Source Record</h4>
<p><strong>A Group Source Record consists of a count of the animals in the Group Source at time <em>t</em>, stratified by sex, age, and health status (e.g., healthy, injured, sick, or dead)</strong>. Other properties of the Group Source Record are Observed Anomalies, Potential Causes of Disease or Death if they apply at time <em>t</em>, among others (see Data Dictionary). Properties of a Group Source Record are documented at the group level, therefore, multiple options can be reported. For example, a Group Source with three dead animals can receive several potential causes of death. <strong>It is not possible to know the distribution of the potential causes of death and other properties across individuals of a Group Source Record</strong>, but only that they were present in at least one individual.</p>
<p><strong>Group Source Records can include a mix of dead, diseased, poisoned, infected, injured, and healthy individuals of the same species or just one of them if all the animals of the species are dead</strong>. A Group Source Record of only healthy animals can also be part of a Health Event depending on the definition of Event.</p>
<p><strong>Group Source Records can have a single individual</strong>. For example, rangers patrolling a protected area might find a dead animal of species X and two dead animals of species Y at the same site (Event) and rangers record their animal data as Group Sources. If this is the case, the single animal of species X belong to a Group Source Record and the two animals of species Y correspond to another Source Group Record. Moreover, if only a single animal of a known herd is observed at time <em>t</em>, then that individual is the representation of the herd for the Source Group Record.</p>
<p><strong>An Event can have more than one Group Source Record of the same species</strong>. For example, animals of the same species in two cages hold by a vendor in market where the vendor is considered an Event. Then the animals per cage could be a unit, and therefore, two Group Source Records of the same species under the same vendor (same Event).</p>
<p>An Group Source can be used directly for a Diagnostic (e.g., e.g., assessment of fat in carcasses of animals belonging to a specific herd). The data model can accommodate Diagnostics applied to a group itself rather than a Specimen taken from that Group Source.</p>
<p><strong>An animal of species X documented as part of a Group Source Record must not be included as an Animal Source Record and vice-versa</strong>. If an Event contains a Group Source Record of species X and an Animal Source Record of species X, then the total number of animals of species X at the Event is the sum of the number of animals of species X in the Group Source Record and the single individual of species X recorded as an Animal Source Record. An example is a herd of cows illegally raised in a protected area whose health is assessed at time <em>t</em>. The herd of this example contains 20 cows. Two of them are sampled and the rest are part of a single unit. In this case, the sampled cows can be added as two Animal Source Records (one per sampled cow) and the remaining 18 animals can be added as a single Group Source Record that distributes the animals per sex, age, and health status. The total number of cows are the individuals in the Group Source Record (18) and the two cows recorded as Animal Records. The “herd” identity of the 20 cows can be kept using a cluster (see “Complexities”). If the Group Source Record contains the original 20 cows and also two Animal Source Records, then the total number of cows will be 22.</p>
</section>
</section>
<section id="animal-source" class="level3">
<h3 class="anchored" data-anchor-id="animal-source">Animal Source</h3>
<p><strong>An Animal Source represents an individual animal whose specific individual-level data is of interest</strong>. Animal Sources can be observed, captured, tested, and provide Specimens at time <em>t</em>, including the full carcass to have a Necropsy completed. Animal Source data includes the Species, the Animal Source ID, the Cross Reference ID, the Sex, among others (see Data Dictionary). Past marking codes (if any have been used) are considered immutable and entered as data of the Animal Source.</p>
<p>An example of Animal Sources are animals of the same or different species caged in a live market. If the data of the animals is to be collected at the individual-level, then each individual in the cage represents an Animal Group Source.</p>
<section id="animal-source-record" class="level4">
<h4 class="anchored" data-anchor-id="animal-source-record">Animal Source Record</h4>
<p><strong>An Animal Source Record consists of an Animal Source at time <em>t</em> </strong>. Properties of an Animal Source Record are its Age, Health Status, Observed Anomalies, and Potential Cause of Disease or Death if they apply at time <em>t</em>, among others (see Data Dictionary). The current marking code of an individual animal at time <em>t</em> is considered mutable and a property of the corresponding Animal Source Record. Properties of an Animal Source Record can receive single or multiple choices. For example, the observed health status is only one (live healthy, live sick) but an Animal Source Record can receive several observed anomalies if they observed in the same individual at time <em>t</em> (wounded, hair loss, diarrhea, etc. See Data Dictionary).</p>
<p>An Animal Source can be either dead, diseased, poisoned, infected, injured, or healthy at time <em>t</em>. <strong>An Animal Source Record with an observed health status “live healthy” can be part of an Event depending on the Event definition</strong> (e.g., a healthy animal of species X next to dead animals of species X, Y, and Z; or healthy animals captured for Specimen collection).</p>
<p>An Animal Source Record can be used directly for a Diagnostic (e.g., X-rays in a live animal at time <em>t</em>). The data model can accommodate Diagnostics applied to the individual itself rather than a Specimen taken from that Animal Source.</p>
<p><strong>An animal of species X documented as part of a Group Source Record cannot be included as an Animal Source Record and vice versa</strong>. If an Event contains a Group Source Record of species X and an Animal Source Record of species X, then the total number of animals of species X at the Event is the sum of the number of animals of species X in the Group Source Record and the single individual of species X recorded as an Animal Source Record. An example is a herd of cows illegally raised in a protected area whose health is assessed at time <em>t</em>. The herd of this example contains 20 cows. Two of them are sampled and the rest are part of a single unit. In this case, the sampled cows can be added as two Animal Source Records (one per sampled cow) and the remaining 18 animals can be added as a single Group Source Record that distributes the animals per sex, age, and health status. The total number of cows are the individuals in the Group Source Record (18) and the two cows recorded as Animal Records. The “herd” identity of the 20 cows can be kept using a cluster (see “Complexities”). If the Group Source Record contains the original 20 cows and also two Animal Source Records, then the total number of cows will be 22.</p>
</section>
<section id="vaccination" class="level4">
<h4 class="anchored" data-anchor-id="vaccination">Vaccination</h4>
<p>The data model allows the inclusion of vaccinations completed in an Animal Source by the time it was captured or immobilized at time <em>t</em>. It is possible to include as many vaccination records as needed to an Animal Source Record. Thus, the vaccination data accumulates for the corresponding Animal Source.</p>
</section>
<section id="carcass" class="level4">
<h4 class="anchored" data-anchor-id="carcass">Carcass</h4>
<p>The carcass of a dead Animal Source can provide Specimens. A Carcass entered to the database comes from a specific Animal Source and they are collected at a single specific time <em>t</em>. Therefore, each Carcass is associated with one and only one Animal Source Record (when the Carcass was collected). Properties of a Carcass include its Decomposition Condition, Storage During Transport, Owner, Availability, among others. Moreover, Carcasses are stored and changes in the storage or movement of the Carcass from one storage facility to another can be multiple over time. Similarly, Carcasses can be exported multiple times. Changes in storage associated with an exportation must occur after the exportation is completed and the Carcass can be stored in the destination facility.</p>
<p>A Carcass does not provide Specimens directly but through the dead Animal Source Record.</p>
<p>A Carcass is not used directly for a Diagnostic but through the dead Animal Source Record (e.g., X-rays in a dead animal). The diagnostic information is part of the set of Diagnostics conducted in the corresponding Animal Source.</p>
<section id="necropsy" class="level5">
<h5 class="anchored" data-anchor-id="necropsy">Necropsy</h5>
<p>A Necropsy is associated with a specific Animal Source Carcass. <strong>The earliest date a Necropsy can occur is when the dead animal was found and a Field Necropsy was conducted (no Carcass collection) or after the Carcass was collected</strong>.</p>
<p>Necropsy data includes the a Necropsy Identifier, the Necropy Cross Reference Identifier, the Date of the Necropsy, findings per system, among others (see Data Dictionary).</p>
<p>Necropsies can be primary or secondary. A primary necropsy usually starts with an intact carcass (not previously used in a necropsy). A secondary necropsy is usually conducted by a veterinarian pathologist using pictures of the primary necropsy or using a carcass already necropsied.</p>
<p>A Necropsy does not provide Specimens directly but through the Animal Source Record.</p>
</section>
</section>
</section>
<section id="environmental-source" class="level3">
<h3 class="anchored" data-anchor-id="environmental-source">Environmental Source</h3>
<p><strong>An Environmental Source is a unit in space where Specimens that cannot be associated with a Group Source, an Animal Source, or Arthropod Source can be collected from</strong> (e.g., the site where feces of unknown source are found). Properties of an Environmental Source include the Environmental Source ID, the Cross Reference ID, among others (see Data Dictionary).</p>
<section id="environmental-source-record" class="level4">
<h4 class="anchored" data-anchor-id="environmental-source-record">Environmental Source Record</h4>
<p><strong>An Environmental Source Record is the biotic or abiotic tissue from the Environmental Source at time <em>t</em> recovered using a common Collection</strong>. For example, the water collected from a pond (Event) at site X (Environmental Source) using a device (Collection) to obtain water at time <em>t</em> (Environmental Source Record). Properties of an Environmental Source Record include Record Number, Type of Tissue, the Quantity, and the Quantity Unit (see Data Dictionary).</p>
<p><strong>A key difference between the Environmental Source and the Group and Animal Sources is that an Event can contain multiple Records of a single Environmental Source (many Collections from the same Environmental Source at time <em>t</em>). Instead, an Event can contain a single Record of many Group and Animal Sources (in one or many Collections)</strong>. An Event can only contain a single Environmental Source, but many Records can be retrieved (one per collection). On the opposite, many bats can be captured in a mist net (many Sources), but they can only provide one Record per Event (see figure at the end of this section).</p>
<p><strong>Another key difference is that “species” for Group and Animal Sources is a property of the Source, whilst for Environmental Sources is a property of the Record</strong>. For example, it could be possible to identify the species that dropped feces found in the field as “mammal”. If this is the case, then the Species property can be completed with this level of taxonomy.</p>
<p><strong>If there is a failed Collection of tissue at time <em>t</em> from the Environmental Source) then it is the Collection that exists without the Environmental Source Record. But an Environmental Source Record cannot be empty</strong>.</p>
</section>
</section>
<section id="arthropod-source" class="level3">
<h3 class="anchored" data-anchor-id="arthropod-source">Arthropod Source</h3>
<p><strong>An Arthropod Source is a unit in space where arthropods can be taken from</strong> (e.g., a household in the forest where traps can be set). Properties of an Arthropod Source include the Source ID, the Cross Reference ID, the Cross Reference ID Origin, among others (see Data Dictionary).</p>
<p><strong>If the interest of the Surveillance Activity is at the arthropod individual level (e.g., butterflies with problems in their wings or with parasites), then the user should consider these arthropods as Animal Sources. Arthropods from Animals Sources (attached ticks, lice, fleas, mites) are Specimens (see next section) from an Animal Source and not Arthropod Sources</strong>.</p>
<section id="arthropod-source-record" class="level4">
<h4 class="anchored" data-anchor-id="arthropod-source-record">Arthropod Source Record</h4>
<p><strong>An Arthropod Source Record is the arthropods of the same species collected at the Arthropod Source at time <em>t</em> using a common Collection</strong>. For example, the mosquitoes of species X collected (Event) at the household in the forest (Arthropod Source) at time <em>t</em> (Arthropod Source Record) using a specific type of trap (Collection). This means that <strong>mosquitoes of a specific species collected by CO2 traps deployed at the Arthropod Source at time <em>t</em> </strong> is an Arthropod Source Record and that mosquitoes of a specific species collected by any number of BG traps deployed at the Arthropod Source at time <em>t</em> is another Arthropod Source Record. The properties of an Arthropod Source Record include Record Number, Species, and the mosquitoes distributed by age, sex, and condition (females only), among others (see Data Dictionary).</p>
<p><strong>Similar to Environmental Sources, a key difference between the Arthropod Source and the Group and Animal Sources is that an Event can contain multiple Records of a single Arthropod Source (many Collections and many species at the same Arthropod Source at time <em>t</em>). Instead, an Event can contain a single Record of many Group and Animal Sources</strong>. An Event can only contain a single Arthropod Source, but many Records can be retrieved (one per Collection per species). On the opposite, many bats can be captured in a mist net (many Sources), but they can only provide one Record per Event (see Figure at the end of the section).</p>
<p><strong>Similarly, another key difference is that “species” for Group and Animal Sources is a property of the Source, whilst for Arthropod Sources is a property of the Record</strong>. For example, mosquitoes obtained through CO2 traps (Collection) deployed at a specific site (Source) will be identified and counted by species after the Collection and they will constitute an Arthropod Source Record.</p>
<p><strong>If there is a failed Collection of arthropods at time <em>t</em> from the Arthropod Source) then it is the Collection that exists without Arthropod Source Records. But an Arthropod Source Record cannot be empty</strong>.</p>
<p><img src="Pictures_and_diagrams/Picture23.png" class="img-fluid" width="600"></p>
</section>
</section>
</section>
<section id="specimen" class="level2">
<h2 class="anchored" data-anchor-id="specimen">Specimen</h2>
<p><img src="Pictures_and_diagrams/Picture9.png" class="img-fluid" style="width:100.0%"></p>
<!-- # Specimens {.unnumbered} -->
<section id="generalities-1" class="level3">
<h3 class="anchored" data-anchor-id="generalities-1">Generalities</h3>
<p>In the data model, Specimens are tissue or material used to conduct Diagnostics (See Diagnostics) and identify hazards or other potential health threats. <strong>Specimens can be collected from Source Records or created from other Specimens (Pooled Specimens) or from products derived of Diagnostic tests (Diagnostic Products). Specimens can contain a single type of tissue coming from a single or multiple Source Records and Specimens, or multiple types of tissue coming from a single or multiple Source Records, Specimens, and Diagnostic Products (see Diagnostics below)</strong>:</p>
<p><img src="Pictures_and_diagrams/Specimens.png" class="img-fluid" style="width:100.0%"> In summary:</p>
<ul>
<li><p>Specimens can be collected from a single Group or Animal Source Records. These Specimens can contain a single type of tissue (e.g., blood) or multiple types of tissue from the same Source Record (e.g., blood and saliva).</p></li>
<li><p>Specimens can be created from a single Environmental and Arthropod Source Records. These Specimens can contain a single type of tissue only (e.g., a site with water can only provide water and a group of <em>Culex sp.</em> mosquitoes can only provide a subset of <em>Culex sp.</em> mosquitoes).</p></li>
<li><p>Specimens can be created by mixing the tissue sourced by many Group, Animal, Environmental, and Arthropod Source Records. Depending on the type of Source Records involved and the tissues provided by each one of them, these Specimens can contain a single type of tissue or several of them.</p></li>
<li><p>Specimens can be created from Diagnostic Products.</p></li>
<li><p>Pooled Specimens with a single or multiple types of tissue can be created by mixing Specimens from a single or multiple Source Records with a single or multiple types of tissue, Specimens from Diagnostic Products, and other Pooled Specimens with a single or multiple types of tissue.</p></li>
</ul>
<p>Specimens are characterized by Specimen ID, Tissue Type, Specimen Original Amount, Specimen Current Amount, their Origin, Ownership, among others (see Data Dictionary). Moreover, Specimens are stored and changes in the storage or movement of the Specimens from one storage facility to another can be multiple over time. Similarly, Specimens can be exported multiple times. Changes in storage associated with an exportation must occur after the exportation is completed and the Specimen can be stored in the destination facility.</p>
<p>Specimens without any amount of tissue left remain in the data model so the last storage, exportation, and use in Diagnostics can be traced.</p>
</section>
<section id="specimens-of-group-source-records" class="level3">
<h3 class="anchored" data-anchor-id="specimens-of-group-source-records">Specimens of Group Source Records</h3>
<p>An example of Group Source Specimen are feces collected from the bottom of a cage with animals of the same species but it is unknown which of the animals dropped the feces.</p>
<p><strong>A Group Source used directly for a Diagnostic (e.g., assessment of fat in carcasses of animals belonging to a specific herd) is not a Specimen (see Sources And Source Records)</strong>.</p>
<p><strong>Group Source Specimens cannot be OBTAINED after the last date the Group Source was observed (the last Record of the Group Source)</strong>. For example, it is possible to collect feces from a cage that restrains a Group Source longitudinally and also from the same cage after the animals of the Group Source were moved. In this case, the Specimen from the empty cage does not belong to the Group Source but to an Environmental Source. The sampling of the feces from the empty cage belongs to a different Event. To keep the connection between the corresponding Specimens, it is possible to cluster the Group Source Records and the Environmental Source Record (see Complexities).</p>
</section>
<section id="specimens-of-animal-source-records" class="level3">
<h3 class="anchored" data-anchor-id="specimens-of-animal-source-records">Specimens of Animal Source Records</h3>
<p>An example of Animal Source Specimen is 2 ml of blood taken from a lion.</p>
<p><strong>An Animal Source used directly for a Diagnostic (e.g., ultrasound in an animal) is not a Specimen (see Sources And Source Records)</strong>.</p>
<p><strong>Animal Source Specimens cannot be OBTAINED after the last date the Animal Source was observed (the last Record of the Animal Source)</strong>. For example, it is possible to collect feces from a cage that restrained an Animal Source longitudinally and also from the same cage after the animal was moved. In this case, the Specimen from the empty cage does not belong to the Animal Source but to an Environmental Source. The sampling of the feces from the empty cage belongs to a different Event. To keep the connection between the corresponding Specimens, it is possible to cluster the Animal Source Records and the Environmental Source Record (see Complexities).</p>
<p><strong>The data model DOES accept Animal Source Specimens that are CREATED after the last date the Animal Source was documented (the last Record of the Animal Source)</strong>. New Specimens can be generated during a Necropsy or from a stored Carcass. In this case, the date of Specimen creation is not necessarily the date when the Animal Source was found, when the Carcass was collected, or when the animal died, but after the storage of the Carcass or the date of the Necropsy (primary or secondary). It is possible to track if a Specimen was collected in the field (from the animal, carcass, during a field necropsy, or the ground near the animal), or in a facility from the carcass or during a primary or secondary Necropsy based on the information entered for Specimens.</p>
</section>
<section id="specimens-from-environmental-source-records" class="level3">
<h3 class="anchored" data-anchor-id="specimens-from-environmental-source-records">Specimens from Environmental Source Records</h3>
<p>An example of Environmental Source Specimen is 10 grams of soil collected from a specific site (Environmental Source). Another useful example are feces of unknown origin found in the field at time <em>t</em> (Source Record) at a specific site (Source).</p>
<p>In this second example it is possible to: i) collect the full feces or ii) take swabs from it. In the former case, the feces are the Source Record and the Specimen (the Specimen can be stored. The Source Record cannot be Stored). In the latter, the feces are just the Source Record whilst the swabs are the Specimens.</p>
<p>In the case of <strong>Environmental Source Records, their type of tissue is determined by the type of the Specimen Source Record</strong>. For example, water collected at time <em>t</em> (Environmental Source Record) from a pond (Environmental Source) can only yield tissue of type “water”. If a Specimen contains tissue from a single Environmental Source Record, then, the type of tissue of the Specimen must be the same type of tissue of the Environmental Source.</p>
<p><strong>The data model DOES accept Environmental Source Specimens that were CREATED after the last date the Environmental Source was visited (the last Record of the Environmental Source)</strong>. New Specimens can be generated from a contained Environmental Source Record (for example, a bottle with sediment that is divided in Specimens later). In this case, the date of Specimen creation is not necessarily the date when the Environmental Source Record was obtained but later.</p>
</section>
<section id="specimens-from-arthropod-source-records" class="level3">
<h3 class="anchored" data-anchor-id="specimens-from-arthropod-source-records">Specimens from Arthropod Source Records</h3>
<p>An example of Arthropod Source Specimen is the set mosquitoes of the same species coming from the same Collection. These are basically sub-groups of the mosquitoes present in the corresponding Source Record. In vector-borne disease surveillance, this last type of Specimen is usually called “pools”.</p>
<p>In the case of <strong>Arthropod Specimens, their type of tissue is determined by the Arthropod Source Record</strong>. If a Specimen contains tissue from a single Arthropod Source Record, then, the type of tissue of the Specimen is “arthropod” by default. In this case, the links of the Specimen with its origin allows to track the species of arthropod involved.</p>
<p><strong>The data model DOES accept Arthropod Source Specimens that were CREATED after the last date the Arthropod Source was visited (the last Record of the Environmental Source)</strong>. New Specimens can be generated from an Arthropod Source Record (for example, taking a new sub-set of mosquitoes from the Source Record and creating a new Specimen with mosquitoes of the species). In this case, the date of Specimen creation is not necessarily the date when the Arthropod Source Record was obtained but later.</p>
</section>
<section id="specimens-with-a-mix-of-tissues-coming-from-a-unique-or-several-source-records" class="level3">
<h3 class="anchored" data-anchor-id="specimens-with-a-mix-of-tissues-coming-from-a-unique-or-several-source-records">Specimens with a mix of tissues coming from a unique or several Source Records</h3>
<p>A bat is swabbed in its oral cavity and in its rectum but then the swabs are placed together in a tube and considered a single Specimen. Then, there is a single Specimen with a unique Source Record origin and with two types of tissue: “rectal swab” and “oral swab” in a tube.</p>
<p>It is possible also to get blood from two bats and mix it. The data model can accommodate this case because <strong>Specimens can have multiple Source Record origins</strong>. In the example, both bled bats (Source Records) are the origin of a single Specimen with “blood” tissue consisting in mm of blood.</p>
<p>Similarly, <strong>it is possible to generate a Specimen by mixing tissue of different type from different Source Records</strong>. For example, the blood (tissue) of a bat captured at time <em>t</em> (Animal Source Record) with the feces (tissue) collected from the bottom of the roost of the same bat at time <em>t</em> (Group Source Record). In this case, the mixed tissue (blood from an Animal Source and Feces from a Group Source) can be included in the data model as a single Specimen with two Source Records as origin and two types of tissue.</p>
<p><strong>The key here is that Specimens are generated by mixing tissue coming from different Source Records, which is different than mixing Specimens</strong>. In the data model, mixing Specimens means mixing data Units already documented (see next).</p>
</section>
<section id="specimens-from-diagnostic-products" class="level3">
<h3 class="anchored" data-anchor-id="specimens-from-diagnostic-products">Specimens from Diagnostic Products</h3>
<p>Products generated by a Diagnostic method (see Diagnostic Products as Specimens below) can be stored and used as Specimen in further Diagnostics (e.g., use cDNA created as part of Diagnostic A used in a new RT-PCR, Diagnostic B). The data model can accommodate Diagnostic Products to be used as Specimens in future Diagnostics. In this case, the origin of these new “Specimens” are specific Diagnostic and not Source Records, and their type is a diagnostic product such as cDNA. The remaining properties of a Specimen from a Diagnostic Product are the same as for Specimens from Source Records (see Data Dictionary).</p>
<p>Diagnostic Products must be added to the data model as a Specimen, so this Specimen can be pooled with other Specimens.</p>
</section>
<section id="pooled-specimens" class="level3">
<h3 class="anchored" data-anchor-id="pooled-specimens">Pooled Specimens</h3>
<p>Specimens can be created by mixing other Specimens of any origin. For example, mixing a Specimen from an Arthropod Source, a Specimen from an Animal Source, a Specimen from an Environmental Source, and a Specimen from an Arthropod Source (see below). The origin of Pooled Specimens is tracked in the data model. In the example, the origin of the Specimen is four Specimens and the new Specimen has potentially four types of tissue. The remaining properties are the same as for Specimens from Source Records (see Data Dictionary).</p>
</section>
<section id="specimens-in-containers" class="level3">
<h3 class="anchored" data-anchor-id="specimens-in-containers">Specimens in Containers</h3>
<p>In situations where space or materials are limited, it is possible that multiple Specimens are stored in a unique container. This approach is clearly not ideal because it can lead to cross-contamination and make actual Specimen tracing more complex. However, the data model has Container Identifier as a property of Specimens (see Data Dictionary). Specific properties for each individual Specimen within the container, such as type, quantity, etc, should allow their visual identification within the container.</p>
</section>
</section>
<section id="diagnostics" class="level2">
<h2 class="anchored" data-anchor-id="diagnostics">Diagnostics</h2>
<p><img src="Pictures_and_diagrams/Picture10.png" class="img-fluid" style="width:100.0%"></p>
<p><strong>Diagnostics in the data model encompass various techniques used to identify hazards (biological, chemical, physical) or physiological problems in either Records of Animal and Group Sources or Specimens obtained from Source Records of any type</strong>. Diagnostics conducted directly in Group and Animals Sources can include general body condition (Group or Animal Source), the width of body fat (Group or Animal Source), an ultrasound (Animal Source). Diagnostics can range from advanced techniques such as metagenomics to basic observations of body condition. Whatever the Diagnostic methods used are, they must be reported in the metadata of the Surveillance Activity.</p>
<p>In the data model, Diagnostics can be conducted in a Laboratory or similar (see Laboratory below) but it is also possible to include field-based assays. Diagnostics can also be conducted by an individual (ranger, hunter, researcher, biologist, etc.) when they involve external or simple observations such as the body condition of an animal to assess a nutrition related hazard.</p>
<p>In the data model, it is considered that each Diagnostic is designed for a specific targeted hazard, such as a viral family or a particular virus species and, therefore, each Diagnostic - Hazard is a Diagnostic method.</p>
<p>Diagnostic properties include the type, method, and result. The type of Diagnostic refers to a general category of diagnostic techniques, such as histopathology, serology, imaging, molecular, clinical. Within each type, there are specific methods available, such as biopsy, ELISA assays, X-rays, PCR, or the width of fat in a herd, respectively. In the data model, <strong>each Diagnostic has a unit of measurement that can either be qualitative or quantitative. The result of the diagnostic is reported as a value of those units</strong>. For example, an agglutination test can report the minimum dilution that causes observable agglutination. A PCR can be reported as presence of bands compatible with the targeted genome sequence or genetic sequence compatible with targeted organism if sequencing followed the genetic sequence amplification. <strong>The interpretation of a Diagnostic (positive, negative, undetermined) corresponds to an “Interpretation” of the Diagnostic</strong> (see Diagnostic Interpretation). Other properties of a Diagnostic include the Diagnostic ID, Diagnostic Code, Diagnostic Cross Reference ID, Diagnostic Cross Reference ID Origin, among others (see Data Dictionary).</p>
<!-- **It is ideal to include supporting information to each Diagnostic, such as digital images or files to corroborate the entered result in the database**.  -->
<section id="diagnostic-products-as-specimens" class="level4">
<h4 class="anchored" data-anchor-id="diagnostic-products-as-specimens">Diagnostic Products as Specimens</h4>
<p>In the data model, <strong>Diagnostics can produce “Diagnostic Products” that are then considered a Specimen</strong>. Diagnostic products as Specimens can be traced with respect to their storage, the quantities available, and any exportation process (See Storage and Export below). Furthermore, Specimens from Diagnostics Products can be used as in other Diagnostics (see Specimens above). For example, cDNA used in a new PCR essay. Basically, the data model accepts a new Specimen generated from a Diagnostic Product with the corresponding type and the origin of ths Specimen is a Diagnostic instead of one or multiple Source Records or other Specimens (See Specimens above).</p>
<!-- ## Diagnosis -->
<!-- ![](Pictures_and_diagrams/Picture11.png){width=100%} -->
</section>
</section>
</section>
<section id="complexities" class="level1">
<h1>Complexities</h1>
<!-- ## Storage -->
<!-- ## Export -->
<section id="laboratories" class="level2">
<h2 class="anchored" data-anchor-id="laboratories">Laboratories</h2>
<p>The data model includes Laboratory entities. Laboratories can conduct Diagnostics to test for hazards in Specimens or in full Source Records (Field-based Diagnostics or simple Diagnostics conducted by people are also possible in the data model. See Diagnostics). Laboratory properties include address, manager, name, Laboratory ID, among others (Data Dictionary). It is also possible to store data regarding Laboratory capabilities in terms of diagnostic tests and storage, and their certifications (Bio safety levels, etc.)</p>
</section>
<section id="interpretation" class="level2">
<h2 class="anchored" data-anchor-id="interpretation">Interpretation</h2>
<section id="interpretation-of-a-hazards-detection-in-a-diagnostic" class="level4">
<h4 class="anchored" data-anchor-id="interpretation-of-a-hazards-detection-in-a-diagnostic">Interpretation of a Hazard’s detection in a Diagnostic</h4>
<p>In the data model, Interpretation of a Diagnostic is the conclusion regarding the detection of a hazard using a Diagnostic targeting that specific hazard, based on a case definition for a positive, negative, or undetermined result provided in the Surveillance Activity metadata. Each Diagnostic receives one Interpretation only.</p>
</section>
<section id="interpretation-of-a-hazards-presence-in-a-specimen" class="level4">
<h4 class="anchored" data-anchor-id="interpretation-of-a-hazards-presence-in-a-specimen">Interpretation of a Hazard’s presence in a Specimen</h4>
<p>In the data model, Interpretation of a Hazard in a Specimen is the conclusion regarding the presence or absence of a specific hazard in a Specimen, based on a case definition for a positive, negative, or undetermined Specimen provided in the Surveillance Activity metadata. The interpretation follows the results from the Diagnostic(s) conducted using the Specimen and the corresponding Diagnostic(s) Interpretation(s). Specimens can receive multiple Interpretations if they are used to conduct several Diagnostics to assess several hazards or if the same hazard is assessed more than once in different diagnostics.</p>
</section>
<section id="interpretation-of-a-hazards-presence-in-a-source-record" class="level4">
<h4 class="anchored" data-anchor-id="interpretation-of-a-hazards-presence-in-a-source-record">Interpretation of a Hazard’s presence in a Source Record</h4>
<p>In the data model, Interpretation of a Hazard in a Source Record is the conclusion regarding the presence or absence of a specific hazard in a Source Record, based on a case definition for a positive, negative, or undetermined Source Record provided in the Surveillance Activity metadata. The interpretation follows the results of the Diagnostic(s) conducted using the Source Record, the Source Record Specimen(s), the corresponding Diagnostic(s), anomalies in the Source Record at a Event, and Necropsy findings.</p>
<p>Source Records can receive multiple Interpretations if multiple hazards are assessed in them or the same hazard is assessed more than once. Sources can have many Interpretations through several Source Records if multiple hazards are assessed in them at time <em>t</em> (Source Record), if the same hazard is assessed more than once at time <em>t</em> (Source Record), or if different or the same hazard are assessed over time (several Source Records of the same Source over time).</p>
</section>
</section>
<section id="clustering-of-locations-events-and-source-records" class="level2">
<h2 class="anchored" data-anchor-id="clustering-of-locations-events-and-source-records">Clustering of Locations, Events, and Source Records</h2>
<section id="clusters-in-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="clusters-in-surveillance-activities">Clusters in Surveillance Activities</h4>
<p>The units <strong>Location, Event, and Source Record could be enough to record the data structure of a specific Surveillance Activity</strong> gathering information during patrols (Field Visits) in Protected Areas (Locations) at points (Events) where dead animals (Source Records) are found at any given time. But another Surveillance Activity could have the following data structure:</p>
<ul>
<li>Visit (<strong>Field Visit</strong>)
<ul>
<li>Protected area (Spatial cluster level 1),
<ul>
<li>Zones within protected area (<strong>Location</strong>),
<ul>
<li>Grid cells within each zone (Spatial cluster level 2),
<ul>
<li>Capture site (<strong>Event</strong>)
<ul>
<li>Mist nets at the capture site (<strong>Collection</strong>),
<ul>
<li>Bats captured (<strong>Source Records</strong>)</li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>Another example is surveillance of wild animals in a live markets. One of the potential options to structure these data is:</p>
<ul>
<li>Visit (<strong>Field Visit</strong>)
<ul>
<li>City (Spatial cluster level 1)
<ul>
<li>Neighborhood (Spatial cluster level 2)
<ul>
<li>Market (<strong>Location</strong>)
<ul>
<li>Vendor (Spatial cluster level 3)
<ul>
<li>Stalls (Spatial cluster level 4)
<ul>
<li>Cage (<strong>Event</strong>)
<ul>
<li>Animals (<strong>Source Records</strong>)
<ul>
<li>Rectal Swab (<strong>Specimens</strong>)</li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<!-- Looking at the examples it is easy to understand that **what an Event represents depends on the Surveillance Activity** (See Event section above) as another Surveillance Activity can establish each Vendor as an Event. However, certain guidelines can be followed to promote standardization of Events (see Recommendations for Standardization below).  -->
<p>Looking at the examples above, it is straightforward to understand that <strong>the units Location, Event, and Collection might not be enough to accommodate the data of all Surveillance Activities and other units (“Clusters”) can be needed</strong>.</p>
<p>A second layer of complexity is the potential need for nested Clusters and non-nested Clusters. The last example above corresponds to a series of nested Clusters where the stalls are within vendors and neighborhoods are within cities. But it is also possible to need non-nested spatial Clusters, for example, “Zip Code”. Zip Code A can include portions of cities 1 and 2, and Zip Code B can also include portions of cities 1, 2, and 3.</p>
<p>And a third layer of complexity is the potential need for spatial and temporal clusters. For example, it is possible that the data structure of the live market Surveillance Activity presented above has to be categorized in decades, year, season, season-year, etc. The following example adds the categorization of the data by season, no matter the markets visited or how many years the Surveillance Activity lasts:</p>
<ul>
<li><strong>Season (Temporal cluster 1)</strong></li>
<li>Visit (<strong>Field Visit</strong>)
<ul>
<li>City (Spatial cluster level 1)
<ul>
<li>Neighborhood (Spatial cluster level 2)
<ul>
<li>Market (<strong>Location</strong>)
<ul>
<li>Vendor (Spatial cluster level 3)
<ul>
<li>Stalls (Spatial cluster level 4)
<ul>
<li>Cage (<strong>Event</strong>)
<ul>
<li>Animals (<strong>Source Records</strong>)
<ul>
<li>Rectal Swab (<strong>Specimens</strong>)</li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>Finally, the number of categories within each Cluster can also be different across Surveillance Activities. For example, two Surveillance Activities may include grid cells as one of their Clusters, however, one Surveillance Activity can include grid cells A - R, whilst the another Surveillance Activity can include grid cell A - W (more categories). In summary:</p>
<ul>
<li>Clusters units can be needed</li>
<li>The Clusters across Surveillance Activities can be different</li>
<li>What these Clusters are grouping across Surveillance Activities can be different</li>
<li>The number of categories can be different among Clusters, within and between Surveillance Activities</li>
<li>Nested, Non-nested or both types of Clusters can be needed</li>
<li>Spatial, Temporal or both Clusters types of Clusters can be needed</li>
</ul>
</section>
<section id="clusters-in-the-data-model" class="level4">
<h4 class="anchored" data-anchor-id="clusters-in-the-data-model">Clusters in the Data Model</h4>
<p>To accommodate these needs in data structure, <strong>the data model allows the inclusion of Clusters</strong> between the Source Record and Event (in the absence of Collection) or Collection and Event, Event and Location, and Location and Field Visit. The data model also allows the inclusion of an undetermined number of nested and non-nested Clusters at each of these levels, and the inclusion of spatial and temporal Clusters. <!-- **Environmental and Arthropod Source Records cannot be clustered in units smaller than Event** but the Locations and Events containing them can be clustered.  --></p>
<p><strong>Unavoidably, the number of Clusters, what Clusters represent, what they cluster, the number of categories per Cluster, and the data to be collected from each of these extra units will vary among Surveillance Activities. Therefore, Cluster properties must be reported in the Surveillance Activity metadata. In the data model, the only default property for each Cluster are the identifier, the cross identifier, the origin of the cross identifier, and a description</strong>. Other potential properties must be documented in a separate file (e.g., an excel sheet) with common identifiers to allow joining the clusters with the corresponding data.</p>
<p>An interesting case study is the collection of Specimens from reindeer carcasses when they are found dead and also from the soil underneath each carcass. In this Surveillance Activity, an Event is a site where dead animals (at least one) are found. So, if two or more carcasses are found at the same site, then the Event contains two Animal Sources, but also two Environmental Sources providing soil Specimens. However, the interest of the researchers is to maintain the connection between the soil Specimens collected below each Animal Source and the Specimens from each Animal Source. Basically, they want to keep track of the soil Specimen that was below each Animal Source. In this case, Cluster unit can be used to group the corresponding Soil Specimen and Animal Specimen in two pairs while keeping the four specimens under the same Event.</p>
<!-- Further characteristics of each clustering unit, whether static or time dependent through the Surveillance Activity period, should be prepared in a file, such as an excel sheet, with columns with the identifiers of all relevant units (Location grouping units. Event grouping units that are not the Location, and Source Record grouping units that are not the Event) and  attached it at the corresponding level. A recommendation is to prepared a sheet for the features of all spatio temporal grouping units visited during a Field Visit and attach it at this level (Field Visit). The columns with the identifiers will allow joining the data of the corresponding Field Visit (Locations, Events, Source Records, grouping units) with the attached and spatial data. -->
</section>
</section>
<section id="surveillance-activities" class="level2">
<h2 class="anchored" data-anchor-id="surveillance-activities">Surveillance Activities</h2>
<p>Initially, it was mentioned that “<strong>Field Visits, Locations, Events, Sources, Source Records, and Diagnostics usually belong to a single Surveillance Activity</strong>. This is the Surveillance Activity that lead to the Field Visits at different Locations to document Events, collect Sources and Specimens, perform Diagnostics for a specific hazard, and provide Diagnosis for Diagnostics, Specimens, and Sources”.</p>
<p>Furthermore, it was established that “<strong>a Surveillance Activity usually includes Field Visits, Locations, Events, Sources, Source Records, and Diagnostics</strong>”.</p>
<p>Here it is explained what are the exceptions to these two general statements and how the data model can handle them.</p>
<section id="field-visits-up-to-specimen-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-up-to-specimen-belong-to-two-or-more-surveillance-activities">Field Visits up to Specimen belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture12.png" class="img-fluid" style="width:100.0%"></p>
<p>Two or more Surveillance Activities used the exact same methodology except for the Diagnostics (they target different hazards but the diagnostics are conducted using the same specimens). For example, the same rodents are trapped and sampled but they are tested for coronaviruses as part of Surveillance Activity 1 and Rickettsia <em>sp</em> as part of Surveillance Activity 2. In this case the Field Visits, Locations, Events, Collections, Source Records, and Specimens, belong to both Surveillance Activities. Instead, the Diagnostics and Diagnosis belong to one of them only.</p>
</section>
<section id="field-visits-up-to-source-records-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-up-to-source-records-belong-to-two-or-more-surveillance-activities">Field Visits up to Source Records belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture13.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when two or more Surveillance Activities overlap in their Field Visits up to Source Records but the Specimens up to Diagnostics are different. For example, the same rodents are trapped and sampled during the same Collection but Surveillance Activity 1 uses rectal swabs to test for coronaviruses whilst Surveillance Activity 2 uses blood for Rickettsia <em>sp</em>. In this case the Field Visits, Locations, Events, Collections, and Source Records belong to both Surveillance Activities. Instead, the Specimens, Diagnostics, and Diagnosis belong to one of them only.</p>
</section>
<section id="field-visits-up-to-collections-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-up-to-collections-belong-to-two-or-more-surveillance-activities">Field Visits up to Collections belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture14.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when two or more Surveillance Activities overlap in their Field Visits up to Collections but the Source Records up to Diagnostics are different. For example, bats are collected using the same mist nets; however, only Megabats are tested for Ebola as part of Surveillance Activity 1. Instead, only Microbats are tested for coronaviruses as part of Surveillance Activity 2. In this case the Field Visits, Locations, Events, and Collections belong to both Surveillance Activities. The Source Records, Specimens, Diagnostics, and Diagnosis belong to one of them only.</p>
</section>
<section id="field-visits-up-to-events-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-up-to-events-belong-to-two-or-more-surveillance-activities">Field Visits up to Events belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture15.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when two or more Surveillance Activities overlap in their Field Visits up to Events, but the Collection up to Diagnostics are different because they have different goals. For example, water is collected from Event A as part of Surveillance Activity 1 and a mosquitos are also collected from Event A as part of Surveillance Activity 2. In this case the Field Visits, Locations, and Events belong to both Surveillance Activities. Instead, the Collections, Source Records, Specimens, Diagnostics, and Diagnosis belong to one of them only.</p>
</section>
<section id="field-visits-and-to-locations-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-and-to-locations-belong-to-two-or-more-surveillance-activities">Field Visits and to Locations belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture16.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when two or more Surveillance Activities overlap in their Field Visits and Locations but the Events up to Diagnostics are different. For example, Surveillance Activity 1 aims to capture bats using mist nets and Surveillance Activity 1 aims to capture rodents. The Field Visit and the Locations could be the same but the capturing of the bats can occur at night and the capture of rodents can occur early morning. In this case the Field Visits and Locations belong to both Surveillance Activities. Instead, the Events, Collections, Source Records, Specimens, Diagnostics, and Diagnosis belong to one of them only.</p>
</section>
<section id="field-visits-belong-to-two-or-more-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="field-visits-belong-to-two-or-more-surveillance-activities">Field Visits belong to two or more Surveillance Activities</h4>
<p><img src="Pictures_and_diagrams/Picture17.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when two or more Surveillance Activities overlap in their Field Visits but the Locations up to Diagnostics are different. Field Visit can be common because it facilitates logistics. For example, coming back from the forest after collecting chimpanzee urine as part of Surveillance Activity 1 and on the way back, a visit is made to a Rehabilitation Center to sample animals as part of Surveillance Activity 2.</p>
<!-- #### Surveillance Activity contains only Diagnostics and Diagnosis -->
<!-- This can happen when Surveillance Activity 2 uses Specimens collected as part of Surveillance Activity 1 to test them again for the same or different hazard. For example, Specimens from bats collected ten years ago will be used in a new study to test them for SARS-CoV-2. The Diagnostics are the only new entity generated as part of Surveillance Activity 2. -->
<!-- ![](Pictures_and_diagrams/Picture18.png){width=100%} -->
</section>
<section id="surveillance-activity-contains-only-specimens-and-diagnostics" class="level4">
<h4 class="anchored" data-anchor-id="surveillance-activity-contains-only-specimens-and-diagnostics">Surveillance Activity contains only Specimens and Diagnostics</h4>
<p><img src="Pictures_and_diagrams/Picture19.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when Surveillance Activity 2 uses Specimens collected as part of Surveillance Activity 1 to test them again for the same or different hazard. For example, Specimens from bats collected ten years ago will be used in a new study to test them for SARS-CoV-2. The Specimen belong to both Surveillance Activities. The Diagnostics are the only new entity generated as part of Surveillance Activity 2, and it is clear that Specimens from Surveillance Activity 1 were used in Surveillance Activity 2.</p>
</section>
<section id="surveillance-activity-contains-only-source-records-specimens-diagnostics-and-diagnosis" class="level4">
<h4 class="anchored" data-anchor-id="surveillance-activity-contains-only-source-records-specimens-diagnostics-and-diagnosis">Surveillance Activity contains only Source Records, Specimens, Diagnostics, and Diagnosis</h4>
<p><img src="Pictures_and_diagrams/Picture20.png" class="img-fluid" style="width:100.0%"></p>
<p>This can happen when Surveillance Activity 2 generates new Specimens from carcasses of Source Records collected as part of Surveillance Activity 1 and tests these new Specimens for a hazard. For example, carcasses of bats collected ten years ago will be used in a new study, Surveillance Activity 2, to get new Specimens and test them for SARS-CoV-2. The Source Records belong to both Surveillance Activities. The new Specimens and Diagnostics are the only new entities generated as part of Surveillance Activity 2, and it is clear that Source Records (the carcasses in this particular case) from Surveillance Activity 1 were used in Surveillance Activity 2.</p>
<!-- Moreover, Necropsies might be conducted as part of a secondary Surveillance Activity (on a Carcass that was collected under other Surveillance Activity) or new Specimens could be created from a previously unused Carcass as part of a new Surveillance Activity. In this case, the new data (Necropsy and Specimens) should receive the second Surveillance Activity only. Diagnostics using the new Specimens should receive the second Surveillance Activity only. The Carcass should receive the Surveillance Activity that lead to its collection only. -->
</section>
<section id="same-source-in-different-surveillance-activities" class="level4">
<h4 class="anchored" data-anchor-id="same-source-in-different-surveillance-activities">Same Source in different Surveillance Activities</h4>
<p>The health of the same animal could be assessed under different Surveillance Activities. For example, a ranger finds a sick animal in a specific date and assess it health and potential cause of disease as part of Surveillance Activity 1. Then, the animal is taken to a rehabilitation center where its health is assessed again at admission and samples are collected for testing as part of Surveillance Activity 2. Finally, the animal health at the rehabilitation center is assessed again three months later and samples are collected again in order to learn what pathogens are circulating within the facility as part of Surveillance Activity 3.</p>
<p>In this case there are three Projects, Surveillance Activities, Field Visits, Events, and Collections independently of each other. There are two Locations. The Animal Source belongs to three Surveillance Activities. The Source Records and Specimens belong to a single Surveillance Activity just like the Diagnostics completed in Surveillance in Surveillance Activity 2 and 3.</p>
<p><img src="Pictures_and_diagrams/Picture21.png" class="img-fluid" style="width:100.0%"></p>
<p>Therefore, Surveillance Activities can be interconnected through common units or the reuse of units generated by previous Surveillance Activities.</p>
</section>
<section id="sources-in-surveillance-activity-but-not-present-in-any-event" class="level4">
<h4 class="anchored" data-anchor-id="sources-in-surveillance-activity-but-not-present-in-any-event">Sources in Surveillance Activity but not Present in any Event</h4>
<p>Studies that track marked individuals over time could succeed in capturing all individually identified animals, so all these Sources are included as Source Records at least once during the study period. However, it is also possible that only a proportion of the individualized Sources are captured or sight during the Surveillance Activity, so these Sources are not present as Source Record of any Event. For the second possibility, it is still important to track the Sources that were part of the sampling frame and censoring them when they were nor part of any Event.</p>
<p>For this reason, the data model can assigned a Surveillance Activity to a Source without including any Field Visits, Location, Event, or Collection. When the study is completed, it will be possible to identify all the Sources involved in the study including those that were never captured (Source 3 in the figure below).</p>
<p><img src="Pictures_and_diagrams/Picture22.png" class="img-fluid" style="width:100.0%"> #### Outbreak Investigation</p>
<p>An outbreak can contains elements associated with two or more Surveillance Activities. For example, the first detection of the outbreak could have been done by a citizen or by rangers patrolling a protected area. The Field Activity, Location, Event, Collection (if any), Source Records, and Specimens collected when the Outbreak is discovered belong to the “Citizen Science” or “Ranger Patrol” Surveillance Activity. However, they also belong to the “Outbreak Investigation” Surveillance Activity, together with Field Activities up to the Diagnostics associated exclusively with the Outbreak Investigation.</p>
<p><img src="Pictures_and_diagrams/Picture24.png" class="img-fluid" style="width:100.0%"></p>
</section>
</section>
</section>
<section id="all-units-in-the-data-model-and-all-relationships" class="level1">
<h1>All Units in the Data Model and All Relationships</h1>
<p>Considering the details explained in the Complexities section, a summary of the relationships in the data model is provided in the following figure (the orange Units and connections apply to Animal Sources only. The red lines apply to Group and Animal Source Records only):</p>
<p><img src="Pictures_and_diagrams/Relationships_All_Sources.png" class="img-fluid" width="680"></p>
<!-- #Recommnedations for Standardization -->


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation column-body">
  <div class="nav-page nav-page-previous">
      <a href="./index.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Introduction</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./data_dictionary.html" class="pagination-link">
        <span class="nav-page-text">Wildlife Health Data Model Dictionary</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>